controllers/groupController.js

```diff
@@ -0,0 +1,81 @@
+import { Group } from '../models';
+import { addUsersToGroup } from '../utils';
+
+export const getAllGroups = async (req, res) => {
+    try {
+        const groups = await Group.findAll();
+        if (groups) {
+            return res.status(200).json(groups);
+        }
+        return res.status(404).send('There are no saved groups');
+    } catch (error) {
+        return res.status(500).send(error.message);
+    }
+};
+
+export const getGroupById = async (req, res) => {
+    try {
+        const { id } = req.params;
+        const group = await Group.findOne({
+            where: { id }
+        });
+        if (group) {
+            return res.status(200).json(group);
+        }
+        return res.status(404).json({ error: 'Group with the specified ID does not exists' });
+    } catch (error) {
+        return res.status(500).send(error.message);
+    }
+};
+
+export const createGroup = async (req, res) => {
+    const { name, permissions } = req.body;
+    const groupObj = { name, permissions: (Array.isArray(permissions) ? permissions : [permissions]) };
```

Comment:
nit: I think using strict type of data is more preferable, so here let's accept only array type

Reply:
@natallia-mikhalenak I use a form in UI to add permissions. There is a corner case for new FormData, if we add only one value it will be a string instead of an array.